validators/cgroup_validator.go

```diff
@@ -81,7 +108,25 @@ func (c *CgroupsValidator) validateCgroupSubsystems(cgroups, subsystems []string
 
 }
 
-func (c *CgroupsValidator) getCgroupSubsystems() ([]string, error) {
+func (c *CgroupsValidator) getCgroupv2Subsystems() ([]string, error) {
+	// Some controllers are implicitly enabled by the kernel.
+	// Those controllers do not appear in /sys/fs/cgroup/cgroup.controllers.
+	// https://github.com/torvalds/linux/blob/v5.3/kernel/cgroup/cgroup.c#L433-L434
+	// We assume these are always available, as it is hard to detect availability.
+	// So, we hardcode the following as "pseudo" controllers.
+	// - devices: implemented in kernel 4.15
+	// - freezer: implemented in kernel 5.2
+	pseudo := []string{"devices", "freezer"}
+	data, err := ioutil.ReadFile(filepath.Join(unifiedMountpoint, "cgroup.controllers"))
+	if err != nil {
+		return nil, err
+	}
+	subsystems := append(pseudo, strings.Fields(string(data))...)
+	return subsystems, nil
+}
+
+func (c *CgroupsValidator) getCgroupv1Subsystems() ([]string, error) {
```

Comment:
getCgroupv1Subsystems -> getCgroupV1Subsystems

Reply:
Fixed it